exercises/F-logical-operators/exercise2.js

```diff
@@ -6,6 +6,27 @@
 */
 
 function isNegative() {}
+function isNegative(number1) {
+  let result = number1 < 0;
+  return result;
+}
+
+function isBetween5and10(number1) {
+  let result = number1 <= 10 && number1 > 5;
+  return result;
+}
+
+function isShortName(name1) {
+  let result = name1.length;
+  if (result < 10) return true;
+  else return false;
+}
```

Comment:
You could also make this function shorter, as it is not necessary to use if statement if the function is returning a boolean. Because result of comparison is a boolean itself. `return name.length < 10` - returns a boolean.